### PR TITLE
Community events

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
Community events are being added to Summit page. It would be good to have them described here as well. 

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1214"></a>

### CODE REVIEW

File: markdown/events/2022/hackathon-october-2022.md [hunk 1]
```diff
@@ -131,6 +131,16 @@ The nf-core website is in desperate need of work, and central tooling always nee
     </table>
 </div>
 
+# Community Streaming Events
+
+Community streaming events are a great way to engage with your local communities while participating in events remotely. By simply booking a room and gathering your colleagues you can create discussion and build your community while enjoying the training, nf-core Hackathon, and/or Nextflow Summit together.
+
+If this is somethign you are interested in you can tell us about your plans using the link below. We will send you extra material to help you prepare and promote your local event. To get started we have a [poster](https://docs.google.com/document/d/1wQDtrVUv1Lro5lZFmeyfaPgJUI2pmGojtL9tWlQiVyQ/edit) that you can download, edit, and share. We have also created a dedicated [community-stream-events](https://nfcore.slack.com/archives/C03Q3RE0RJQ) Slack channel where you can ask questions and share information about your event.
```
Comment (ewels) [inline markdown/events/2022/hackathon-october-2022.md:138]:
```suggestion
If this is somethign you are interested in you can tell us about your plans using the link below. We will send you extra material to help you prepare and promote your local event. To get started we have a [poster](https://docs.google.com/document/d/1wQDtrVUv1Lro5lZFmeyfaPgJUI2pmGojtL9tWlQiVyQ/edit) that you can download, edit, and share. We have also created a dedicated [`#community-stream-events`](https://nfcore.slack.com/archives/C03Q3RE0RJQ) Slack channel where you can ask questions and share information about your event.
```

**Review by ewels — APPROVED** [review]

**Comment by ewels** [comment]
I wonder if the slack channel could have summit in the name and be a bit shorter? But maybe you're trying to generalise or something. Only a thought, happy either way.

File: markdown/events/2022/hackathon-october-2022.md (hunk 1 above)
Comment (christopher-hakkaart) [inline markdown/events/2022/hackathon-october-2022.md:138]:
```suggestion
If this is something you are interested in you can tell us about your plans using the link below. We will send you extra material to help you prepare and promote your local event. To get started we have a [poster](https://docs.google.com/document/d/1wQDtrVUv1Lro5lZFmeyfaPgJUI2pmGojtL9tWlQiVyQ/edit) that you can download, edit, and share. We have also created a dedicated [community-stream-events](https://nfcore.slack.com/archives/C03Q3RE0RJQ) Slack channel where you can ask questions and share information about your event.
```

File: markdown/events/2022/training-october-2022.md [hunk 2]
```diff
@@ -74,3 +74,13 @@ The training will use a preconfigured development environment in [Gitpod](https:
 Click on the link below and fill out the registration form to secure your place :tada:
 
 <a class="btn btn-success btn-lg" href="https://seqera.typeform.com/oct-22-training"><i aria-hidden="true"></i>Register now</a>
+
+# Community Streaming Events
+
+Community streaming events are a great way to engage with your local communities while participating in events remotely. By simply booking a room and gathering your colleagues you can create discussion and build your community while enjoying the training, nf-core Hackathon, and/or Nextflow Summit together.
+
+If this is somethign you are interested in you can tell us about your plans using the link below. We will send you extra material to help you prepare and promote your local event. To get started we have a [poster](https://docs.google.com/document/d/1wQDtrVUv1Lro5lZFmeyfaPgJUI2pmGojtL9tWlQiVyQ/edit) that you can download, edit, and share. We have also created a dedicated [community-stream-events](https://nfcore.slack.com/archives/C03Q3RE0RJQ) Slack channel where you can ask questions and share information about your event.
```
Comment (christopher-hakkaart) [inline markdown/events/2022/training-october-2022.md:82]:
```suggestion
If this is something you are interested in you can tell us about your plans using the link below. We will send you extra material to help you prepare and promote your local event. To get started we have a [poster](https://docs.google.com/document/d/1wQDtrVUv1Lro5lZFmeyfaPgJUI2pmGojtL9tWlQiVyQ/edit) that you can download, edit, and share. We have also created a dedicated [`#community-stream-events`](https://nfcore.slack.com/archives/C03Q3RE0RJQ) Slack channel where you can ask questions and share information about your event.
```

**Pull request merged**